app/assets/javascripts/simulations.js

```diff
@@ -0,0 +1,20 @@
+// # Place all the behaviors and hooks related to the matching controller here.
+// # All this logic will automatically be available in application.js.
+// # You can use CoffeeScript in this file: http://coffeescript.org/
+
+$(document).ready(function() {
+  $('.simulation-next-row').on('click', function(event) {
+    tableBody = $('#simulation-table')[0]
+    row = tableBody.children[tableBody.children.length - 1];
+    rowCopy = row.cloneNode(true);
+    _.map(rowCopy.children, function(element){
```

Comment:
@jktin12 For reference, this takes the `name` and `id` attributes of the `select` and/or `input` nodes in the last table row and updates the id number in them.

I plan to replace it with a timestamp later (that was the recommendation from that rails guides).

```javascript
function(element){
  selectElement = element.querySelector('select') || element.querySelector('input');
  id = _.now();
  selectElement.name = selectElement.name.replace(/\d+/g, id);
  selectElement.id = selectElement.id.replace(/\d+/g, id);
});
```